samza-core/src/main/java/org/apache/samza/config/TaskConfig.java

```diff
@@ -136,6 +136,11 @@
       "task.transactional.state.retain.existing.state";
   private static final boolean DEFAULT_TRANSACTIONAL_STATE_RETAIN_EXISTING_STATE = true;
 
+  // Job Elasticity related configs
+  // Take effect only when job.elasticity.factor is > 1. otherwise there is no elasticity
+  private static final String TASK_ELASTICITY_FACTOR = "task.elasticity.factor";
```

Comment:
is this more like a task-to-partition mapping factor?

Reply:
I am looking at this more as a split multiple. As in if the factor = 2 then split each original task into 2 virtual/elastic tasks. It can also be looked at as task-to-partition/ssp factor where factor = 2 means each ssp is read by 2 virtual/elastic tasks. would lend the same semantics.